freqtrade/persistence/models.py

```diff
@@ -116,7 +116,7 @@ class Order(_DECL_BASE):
     ft_order_side = Column(String(25), nullable=False)
     ft_pair = Column(String(25), nullable=False)
     ft_is_open = Column(Boolean, nullable=False, default=True, index=True)
-
+    is_recovered = Column(Boolean, nullable=False, default=False)
```

Comment:
What do you imagine this variable to be used for?

The naming suggests that you're trying to say "this recovered and is positive again" - please keep in mind that this is ONE usecase for this - but the most likely a lot more profitable one is to increase stake in positions which are going for you - where "is_recovered" is simply a wrong name for.

Reply:
what about is_processed?

Reply:
It's not about the name alone - i don't understand what you aim to do with this.